gap/base/methods.gd

```diff
@@ -0,0 +1,40 @@
+#############################################################################
+##
+##  This file is part of recog, a package for the GAP computer algebra system
+##  which provides a collection of methods for the constructive recognition
+##  of groups.
+##
+##  This files's authors include Sergio Siccha.
+##
+##  Copyright of recog belongs to its developers whose names are too numerous
+##  to list here. Please refer to the COPYRIGHT file for details.
+##
+##  SPDX-License-Identifier: GPL-3.0-or-later
+##
+##
+##  Declarations of recog methods
+##
+#############################################################################
+
+#! @Chapter methsel
+#! @Section whataremethods
+#! @Description
+#! TODO: how to get this into the chapter with the GAPDoc label "methsel"?
+#! TODO
+#! method     : the function itself
+#! rank       : an integer rank
+#! stamp      : a string describing the method uniquely
+#! comment    : an optional comment to describe the method for humans
```

Comment:
I assume the above comment is a draft and perhaps not meant for `IsRecogMethod` but rather for `RecogMethod`?